File: C/04 - largestNumber.c
#include <stdio.h>

int main() {
    int a,b,c;
    printf("Enter first number: ");
    scanf("%d", &a);
    printf("Enter second number: ");
    scanf("%d", &b);
    printf("Enter third number: ");
    scanf("%d", &c);
    if (a > b && a > c) { printf("%d is the largest number", a); }
    else if (b > a && b > c) { printf("%d is the largest number", b); }
    else if (c > a && c > b) { printf("%d is the largest number", c); }
    else { printf("At least two largest numbers are equal"); }
    return 0;
}

File: C#/04 - largestNumber.cs
using System;

public class largestNumber
{
    public static void Main(string[] args)
    {
        int a,b,c;
        Console.Write("Enter first number: ");
        a = Convert.ToInt32(Console.ReadLine());
        Console.Write("Enter second number: ");
        b = Convert.ToInt32(Console.ReadLine());
        Console.Write("Enter second number: ");
        c = Convert.ToInt32(Console.ReadLine());
        if (a > b && a > c){ Console.WriteLine($"{a} is the largest number"); }
        else if (b > a && b > c){ Console.WriteLine($"{b} is the largest number"); }
        else if (c > a && c > b){ Console.WriteLine($"{c} is the largest number"); }
        else { Console.WriteLine("At least two largest numbers are equal"); }
    }
}